### PR TITLE
Fixes to make the web store entry look alright

### PR DESCRIPTION
**Pull request by itskdog**
Curent issues:
- Old AdBlock description still in place
- We need a better icon (128x128, not 16x16)
- No short_name in manifest (in case space is too small - "AdBlock with CatBlock" is v. long)

Tomas: I have made you a tester, so that you can access the extension from:  
https://chrome.google.com/webstore/detail/mdcgnhlfpnbeieiiccmebgkfdebafodo/

### CODE REVIEW

**Comment by itskdog** [comment]
I'll deal with the code changes - @tomasko126 can you do the image update?

**Comment by itskdog** [comment]
I've done the first & last points, but am not too sure about the icon.

@tomasko126 or @Jeremyboy222: do either of you know how to make the icon larger, and remove the pixelation? I tried, but couldn't.

**Comment by tomasko126** [comment]
@kpeckett I think we need a new icon.. @Jeremyboy222, as you have experience with photo-editing SW, could you make a new logo for CatBlock?

**Comment by JeremyKeusters** [comment]
The URL doesn't work for me.. Where's the current logo?

**Comment by tomasko126** [comment]
@Jeremyboy222 You need to ask @kpeckett to be a tester & current CatBlock logo is [here](https://github.com/CatBlock/catblock/blob/master/catblock/icons/icon19.png)

**Comment by JeremyKeusters** [comment]
@kpeckett Can I be tester? :p

**Comment by itskdog** [comment]
@Jeremyboy222 I added jeremy.keusters@gmail yesterday

**Comment by JeremyKeusters** [comment]
@kpeckett Thx!

**Comment by itskdog** [comment]
@Jeremyboy222 How's it going? The current logo is in the /catblock/icons folder - are you able to upscale that to 64x64 and 128x128?

**Comment by JeremyKeusters** [comment]
@kpeckett What do you think of this? 
catblock (Please note that this logo is displayed on a white background so you don't see the borders..)

**Comment by JeremyKeusters** [comment]
@kpeckett Here is the same image but this one is on a green background.. It's 2050x2050! :)
catblock_greenbg

**Comment by itskdog** [comment]
Looks nice - @tomasko126 any thoughts?

@Jeremyboy222 just checking, this is based on the current AdBlock logo, right, not the "modern" one, which I don't think has ever been in any public GPL-licenced release.

File: manifest.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "AdBlock with CatBlock",
+	"short_name": "CatBlock",
```
Comment (tomasko126) [inline manifest.json]: @kpeckett Can you fix tabs here?

Comment (itskdog) [inline manifest.json]: @tomasko126 Thanks - fixed!

**Comment by itskdog** [comment]
The link is now dead:

> Dear Developer,
> Your Google Chrome item, "AdBlock with CatBlock," with ID:  
> mdcgnhlfpnbeieiiccmebgkfdebafodo did not comply with our program policies  
> and was removed from the Google Chrome Web Store.
> Your item did not comply with the following section of our policy:
> Don't pretend to be someone else, and don't represent that your app is  
> authorized by or produced by another company or organization if that is not  
> the case. Products or the ads they contain also must not mimic  
> functionality or warnings from a user's operating system or browser.  
> Developers should not divert users or provide links to any other site that  
> mimics Chrome Web Store or passes itself off as Chrome Web Store.

**Comment by itskdog** [comment]
@Jeremyboy222 If/when we have a meeting, it would mostly be about getting the logo sorted. If you have time, getting a draft copy ready would be useful. I would like to have some sort of logo for the release of v1.1.1 - this is the only thing blocking v1.1.1 right now.

**Comment by JeremyKeusters** [comment]
@kpeckett What dimensions do you want then?

**Comment by itskdog** [comment]
@Jeremyboy222 256x256, though having a copy of the PSD (or whatever you use to edit them) would be great as well.

**Comment by JeremyKeusters** [comment]
Here you are, I'll give you the PSD later (can't upload it here in github)
catblock_256

**Comment by itskdog** [comment]
This is the one you remade from scratch, right?

**Comment by JeremyKeusters** [comment]
@kpeckett Well, this is the one where I removed the 'Hand icon' from the AdBlock logo and put a claw onto it..

**Comment by itskdog** [comment]
Can you come online on slack, it will be easier to communicate rather than through here.

**Comment by tomasko126** [comment]
After chat with @kpeckett, we've decided to use @Jeremyboy222's logo as a CatBlock logo.

**Comment by tomasko126** [comment]
@kpeckett @Jeremyboy222 I've created new icons, based on @Jeremyboy222's original icon. Please take a look :)

**Comment by tomasko126** [comment]
@kpeckett @Jeremyboy222 

BTW, I didn't change green icon, which is shown, when domain is whitelisted. Could we display @Jeremyboy222's icon, but with green background instead of red?
Thoughts?

**Comment by itskdog** [comment]
I've already put changing thumbs up to paws up as a goal for v1.2 - if it isn't ready by the time we release v1.1.1, then it will have to wait

**Pull request merged**